devel/jmeter/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       java 1.0
+
+name            jmeter
+categories      devel benchmarks java net
+platforms       darwin
+license         Apache-2.0
+supported_archs i386 x86_64
+maintainers     {outlook.de:judaew} openmaintainer
+
+version         5.3
+java.version    1.8+
+
+description     Apache JMeter - Load testing tool
+long_description \
+    Apache JMeter is a Java application designed to load test functional \
+    behavior and measure performance. It was originally  designed  for  \
+    testing  Web Applications  but  has  since expanded to other test functions.
```

Comment:
```suggestion
    behavior and measure performance. It was originally designed for \
    testing web applications but has since expanded to other test functions.
```